embedded-keycloak/src/main/java/com/playtika/test/keycloak/KeycloakContainer.java

```diff
@@ -41,6 +42,45 @@ protected void configure() {
         withImportFile(properties.getImportFile());
     }
 
+    private void withDB() {
```

Comment:
Codacy found an issue: [The method 'withDB()' has an NPath complexity of 512, current threshold is 200](https://app.codacy.com/manual/PlaytikaCodacy/testcontainers-spring-boot/pullRequest?prid=5969041)